packages/tool-cache/__tests__/tool-cache.test.ts

```diff
@@ -346,6 +347,40 @@ describe('@actions/tool-cache', function() {
         await io.rmRF(tempDir)
       }
     })
+  } else if (IS_MAC) {
+    it('extract .xar', async () => {
+      const tempDir = path.join(tempPath, 'test-install.xar')
+      const sourcePath = path.join(__dirname, 'data', 'archive-content')
+      const targetPath = path.join(tempDir, 'test.xar')
```

Comment:
Please remove this binary, it is still committed above

Reply:
Oh wow, I managed to commit another prebuilt archive when removing the first one 🤦 
Fixed now.